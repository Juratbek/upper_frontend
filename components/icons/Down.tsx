import { IIconProps } from './types';

export const DownIcon = ({ width = 24, height = 24 }: IIconProps) => (
  <svg
    width={width}
    height={height}
    viewBox='0 0 24 24'
    fill='none'
    xmlns='http://www.w3.org/2000/svg'
  >
    <path d='M6 9L12 15L18 9' stroke='#A2A4A5' strokeWidth='1.6' />
  </svg>
);
