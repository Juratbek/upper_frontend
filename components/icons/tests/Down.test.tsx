import { render, screen } from '@testing-library/react';

import { DownIcon } from '../Down';

describe('Down icon', () => {
  it('renders the icon', () => {
    render(<DownIcon />);
    const icon = screen.getByRole('icon');
    expect(icon).toMatchSnapshot();
  });
});
