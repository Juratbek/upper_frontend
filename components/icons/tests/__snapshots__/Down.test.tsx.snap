// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Down icon > renders the icon 1`] = `
<svg
  fill="none"
  height="24"
  role="icon"
  viewBox="0 0 24 24"
  width="24"
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M6 9L12 15L18 9"
    stroke="#A2A4A5"
    stroke-width="1.6"
  />
</svg>
`;
